velox/exec/HashTable.h

```diff
@@ -642,33 +630,31 @@ class HashTable : public BaseHashTable {
   // Adds a row to a hash join table in kArray hash mode. Returns true
   // if a new entry was made and false if the row was added to an
   // existing set of rows with the same key.
-  bool arrayPushRow(char* FOLLY_NULLABLE row, int32_t index);
+  bool arrayPushRow(char* row, int32_t index);
 
   // Adds a row to a hash join build side entry with multiple rows
   // with the same key.
-  void pushNext(char* FOLLY_NULLABLE row, char* FOLLY_NULLABLE next);
+  void pushNext(char* row, char* next);
 
   // Finishes inserting an entry into a join hash table. If the insert
   // would fall outside of 'partitionBegin' ... 'partitionEnd', the
   // insert is not made but the row is instead added to 'overflow'.
   void buildFullProbe(
       ProbeState& state,
       uint64_t hash,
-      char* FOLLY_NULLABLE row,
```

Comment:
Thank you for the refactoring, but it makes it harder to see material changes. Would it be possible to extract refactoring into a separate PR?